mysql-test/suite/tianmu/t/out_of_range_issue1151.test

```diff
@@ -20,10 +20,10 @@ insert into tiny values(128, 0);
 --error 1264
 insert into tiny values(1234, 0);
 # column unsigned b out of value[128, 255], deal with tianmu
---error 1264
-insert into tiny values(0, 128);
---error 1264
-insert into tiny values(0, 255);
+# --error 1264,BUG: https://github.com/stoneatom/stonedb/issues/1236
```

Comment:
Here, some basic test should be added to make sure, anyone's code undermine this feature.